sparkle/src/components/Tree.tsx

```diff
@@ -171,7 +171,7 @@ Tree.Item = function ({
             }}
           />
         )}
-        {type === "leaf" && <div className="s-w-8 s-flex-shrink-0"></div>}
+        {type === "leaf" && <div className="s-w-[34px] s-flex-shrink-0"></div>}
```

Comment:
Let's try to avoid having hardcoded pixels value, let's create [custom ones](https://tailwindcss.com/docs/width#customizing-your-theme) if needed